meetings/2024-01-10.md

```diff
@@ -0,0 +1,125 @@
+  * Second one is to add as experimental, Matteo what did you see with breakage
+    * Matteo, from what I saw breakage was extensive
+    * Robert the only way forward is to add another new API, deprecate next tick to discourage
+      future use.
+  * Can remove from agenda at this point.
+
+* enable corepack by default [#50963](https://github.com/nodejs/node/issues/50963)
+  * Yagiz, have not seen to many changes needed for a while so seems stable, likely that we
+    should make a decision on making it stable or if that is not going to enable by default then
+    we should remove. If we don’t enable by default then we don’t have a path to removing npm.
+  * Matteo, shipping a package manager by default is one of the key elements of what made
+    Node.js successful. Don’t see a good technical reason to migrate them out. Advantage of
+    shipping npm, is on stability of the build. One issue is loss of repeatable builds
+  * Michael, main point is if goal of people in making stable is a step towards removing npm,
+    then that is the discussion we should have versus making it stable and kicking that down
+    the road.
+  * Yagiz, recommend that we either remove npm or add other package managers.
+  * Robert, whatever we do, removing npm is not an option, then leads do we discuss adding
+    package manager managers
+  * Yagiz why can we not remove npm
+  * Geoffrey, would be large breaking change
+  * Robert, not worth it for political issue
+  * Yagiz, should we not reduce the bundle size
+  * Richard, unlikely to get a consensus. We will likely need a vote.
+  * Matteo, propose we remove npm from corepack, then enable corepack by default. If we
+    Have a vote lets vote on removing npm, versus vote on corepack.
+  * Geoffrey, if the goal is to offer bundle size, don’t we offer additional binaries.
+  * Yagiz, if vote is removal of npm it should include alternative of adding more package
+    managers
+  * Robert, does it even makes sense to have corepack if we are not going to remove npm
+  * Matteo, if you enable corepack, then typing pnpm it just works its great. But when using
+    corepack you don’t get a clear picture of the match between Node.js and package manager
+    installed.
+  * Richard, the issue around not knowing what you get. Original version only installed a specific
+    version. The project requested that it pull in latest.
+  * Vote
+    * We agree our goal is to remove npm from Node.js:   Yes/No
+    * If we are not going to remove npm, should corepack be included in Node.js Yes/No
+    * Should we enable corepack by default today: Yes/No
+  * Geoffrey, we don’t necessarily need to rush into decision. Would be good to get how popular
+    is corepack
+  * Ruy, definitely going to be a large breaking change
+  * Geoffrey, the person who wants to champion removal of npm should write up proposal and
+    then we vote on that.
+  * Ruy, has been mentioned that distributing npm as are of corepack might not be legally ok.
```

Comment:
Does corepack actually redistribute npm, or does it just fetch it from the registry?

See related https://github.com/nodejs/corepack/issues/104#issuecomment-1368029808

Reply:
@styfle I think thats a reasonable question, but the notes are meant to record what Ruy say so I don't think we necessarily need to update them to answer the question.

Reply:
Makes sense. I just didn't know where to leave comments at the time. I have since found this https://github.com/nodejs/node/issues/50963